src/components/shared/iconlessTooltip/index.js

```diff
@@ -2,13 +2,11 @@ import React from 'react';
 import Tooltip from '../../toolbox/tooltip/tooltip';
 
 const IconlessTooltip = ({
```

Comment:
It is better this feature implement as a property to the tooltip component instead of the current implementation. then you will have an optional property that lets to hide the icon.

Reply:
Good point. The component wasn't needed. Solved in https://github.com/LiskHQ/lisk-hub/pull/2539/commits/d0077e97bc165dda1feb37e2de3e12dd2fce9f4b